src/components/Books/Books.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import styles from './Books.module.css';
+
+const Books = () => (
+  <div className={styles.main_page}>
+    <ul className={styles.book_section_container}>
+      <li className={styles.book_section}>
+        Sample Book One
+        <br />
+        <br />
+        <button type="button">Remove</button>
+      </li>
+      <li className={styles.book_section}>
+        Sample Book Two
+        <br />
+        <br />
+        <button type="button">Remove</button>
+      </li>
+      <li className={styles.book_section}>
+        Sample Book Three
+        <br />
+        <br />
+        <button type="button">Remove</button>
+      </li>
+    </ul>
```

Comment:
- [ ] As it is a requirement for this project, please kindly add a `BookList` component that;
 - Renders a list of books using the `Book component`,
 - Renders a form by using the `CreateNewBook` component.